python/lsst/obs/base/ingest.py

```diff
@@ -326,10 +352,11 @@ def _determine_instrument_formatter(self, dataId, filename):
                 raise RuntimeError(
                     f"Instrument {dataId['instrument']} for file {filename} not known to registry"
                 ) from e
-            FormatterClass = Formatter
+            FormatterClass = FitsRawFormatterBase
```

Comment:
Looks like I got overzealous after realizing that `Instrument.getRawFormatter` does need to return `FitsRawFormatterBase`, because `DefineVisitsTask` needs its `makeRawSkyWcsFromBoresight`.  I suppose we could consider moving that method to the `Instrument` class itself, instead of strapping new interfaces on formatters.  In any case, you're right that `RawIngestTask` can just use `Formatter` and I'll make the switch.